### ISSUE TITLE
Difficult to know it if You have ordered the correct stuff

### ISSUE THREAD

**Issue by KickiLagerstedt** [Clinical-Genomics/cgadmin#18]
Would it be possible to have a more clarifying way if picking sequencing order. For instance:

"WGS 30x" instead of "WGSPCFC030"
"WGS 30x trio" instead of "WGTPCFC030"
"Reanalysis WES" instead of "EXXCUSR000"

**Comment by KickiLagerstedt** [comment]
The list was very long to choose from and I hope I picked the correct ones..

**Comment by robinandeer** [comment]
Yes, I agree - we should be able to handle this in some clearer way

We will think about it! Thanks for the feedback

**Comment by vwirta** [comment]
Two options, either visualise an alias for the actual application tag as Kicki suggests, or have a search function that limits what is shown in the list to what is actually typed into a text field.
What is technically easier? Or perhaps implement both?
